.github/workflows/pull-request-checks.yml

```diff
@@ -21,9 +21,4 @@ jobs:
         if: github.event.pull_request.user.login != 'dependabot[bot]'
         with:
           checklist-items: |
-            New functionality includes testing.
-            All tests pass
-            New functionality has been documented.
-            New functionality has javadoc added
-            Commits are signed per the DCO using --signoff
-            Commit changes are listed out in CHANGELOG.md file (See: [Changelog](../blob/main/CONTRIBUTING.md#changelog))
+            Functionality includes testing.
```

Comment:
What do you think about removing this check entirely? Seems to me that If every item isn't important to require an 'affirmative check', then none of them are - it isn't worth the cognitive load on contributors. 

I know disabling this check will be popular with based on [1] previous discussions.

- [1] https://github.com/opensearch-project/OpenSearch/issues/10970

Reply:
Sounds good, I'll remove and update this PR.